frontend/public/components/utils/webhooks.tsx

```diff
@@ -42,30 +59,49 @@ export const WebhookTriggers: React.SFC<WebhookTriggersProps> = ({ resource }) =
       : <span className="text-muted">No secret</span>;
   };
 
+  const copyWebhookToClipboard = trigger => {
+    const triggerProperty = getTriggerProperty(trigger);
+    const secretName = _.get(trigger, [triggerProperty, 'secretReference', 'name']);
```

Comment:
What happens when the webhook isn't using a secret reference?

Reply:
was looking into doc and saw anly the `secretReference` but now I see that there is the `secret` field which is [obsolete](https://github.com/openshift/api/blob/master/build/v1/types.go#L949)
WIll update the PR to fix